### PR TITLE
feat(ONYX-375): add ContextModule and OwnerType for alerts filters

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR resolves [ONYX-375]

### Description

Added ContextModule and OwnerType for alerts filters tracking on Eigen

### PR Checklist (tick all before merging)

- [ ] If I've added a new file to the tree I've exported it from the common [`index.ts`](https://github.com/artsy/cohesion/blob/main/src/Schema/Events/index.ts)
- [ ] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [ ] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[ONYX-375]: https://artsyproduct.atlassian.net/browse/ONYX-375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Schema/Values/ContextModule.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export enum ContextModule {
   curatedHighlightsRail = "curatedHighlightsRail",
   curatedTrendingArtistsRail = "curatedTrendingArtistsRail",
   curatorsPicksEmergingRail = "curatorsPicksEmergingRail",
+  currentAlertArtworkMatches = "currentAlertArtworkMatches",
```
Comment (dblandin) [inline src/Schema/Values/ContextModule.ts:73]: **suggestion**: Following up on [this Slack thread](https://artsy.slack.com/archives/C05EQL4R5N0/p1700049456288239), what do you think of `alertDetails`, `alertFilters`, and `alertConfirmation` as new context modules to add here?

File: src/Schema/Values/OwnerType.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@
 export enum OwnerType {
   activities = "activities",
   allArtistSeries = "allArtistSeries",
+  alertConfirmation = "alertConfirmation",
+  alertFilters = "alertFilters",
```
Comment (dblandin) [inline src/Schema/Values/OwnerType.ts:10]: **suggestion**: I don't think we need these values, which map to the specific steps in the alert flow. I think those are better captures as ContextModule values.

Although we might consider additions to eventually replace the existing `savedSearch` and `savedSearches` values such as `alert` and `alerts`.

Comment (dariakoko) [reply src/Schema/Values/OwnerType.ts:10]: Does that mean that you suggest excluding the owner_type from the event at all and use ContextModule instead? Wha value would you suggest to use for the ContextModule in this case?
[dock with the event details](https://docs.google.com/spreadsheets/d/1iBmzvJwgZ21J8w2VbVY3JAbH0EHbvUMz77HnZs3Ht9U/edit#gid=62721100)

Comment (dblandin) [reply src/Schema/Values/OwnerType.ts:10]: Ahh, you're totally right. For the app, we need these new owner types as there are distinct screens to track. In that case don't forget to include the in the `ScreenOwnerType` values below in this same file. While you're in here, maybe adding `alerts` and `alertDetails` as additional `ScreenOwnerType` values?

File: src/Schema/Values/ContextModule.ts (hunk 1 above)
Comment (dariakoko) [reply src/Schema/Values/ContextModule.ts:73]: missed this one
adding now

**Comment by artsyit** [comment]
:rocket: PR was released in [`v4.157.0`](https://github.com/artsy/cohesion/releases/tag/v4.157.0) :rocket:

**Pull request merged**